src/charts/createDotPlot.js

```diff
@@ -39,26 +39,26 @@ export function createDotPlot(this_, d) {
   const chartData = d.statistics.values
     .sort(
       (a, b) =>
-        (a.prop_n > b.prop_n
+        a.prop_n > b.prop_n
           ? -2
-          : a.prop_n < b.prop_n ? 2 : a.key < b.key ? -1 : 1)
+          : a.prop_n < b.prop_n ? 2 : a.key < b.key ? -1 : 1
```

Comment:
Yeah, seems like it, though I'm not setting any settings.  Perhaps it's a version difference?